src/Services/Profile/Profile.Application/UseCases/EducationUseCases/Commands/AddEducationToProfile/AddEducationToProfileHandler.cs

```diff
@@ -30,23 +39,21 @@ public async Task<List<ProfileEducationResponseDto>> Handle(AddEducationToProfil
             throw new NotFoundException("Education", request.Dto.EducationId);
         }
         
-        var isProfileContainsEducation = profileWithEducation.ContainsEducation(request.Dto.EducationId);
+        var isProfileContainsEducation = profile.ContainsEducation(request.Dto.EducationId);
 
         if (isProfileContainsEducation)
         {
             throw new AlreadyContainsException(ExceptionMessages.ProfileContainsEducation);
         }
 
         var profileEducation = _mapper.Map<ProfileEducation>(request.Dto);
-        await _unitOfWork.EducationRepository.AddEducationToProfileAsync(profileWithEducation, profileEducation);
+        await _unitOfWork.EducationRepository.AddEducationToProfileAsync(profile, profileEducation);
```

Comment:
await must be preceeded and followed by empty lines. Same for other awaits

Reply:
you can use static code analyzers for automatic check of this. For example - StyleCop